torch/onnx/symbolic_opset9.py

```diff
@@ -2225,14 +2226,33 @@ def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
     variance = g.op("ReduceMean", pow(g, numerator, two_cst), axes_i=axes)
     denominator = sqrt(g, add(g, variance, eps_cst))
 
-    layer_norm = g.op("Div", numerator, denominator)
+    normalized = g.op("Div", numerator, denominator)
 
     if not (weight is None or symbolic_helper._is_none(weight)):
-        layer_norm = mul(g, layer_norm, weight)
+        normalized = mul(g, normalized, weight)
     if not (bias is None or symbolic_helper._is_none(bias)):
-        layer_norm = add(g, layer_norm, bias)
+        normalized = add(g, normalized, bias)
+
+    if return_mean_rstd:
+        rdenominator = reciprocal(g, denominator)
+        return normalized, mean, rdenominator
+    else:
+        return normalized, None, None
```

Comment:
```suggestion
    return normalized, None, None
```

this branch can be removed and dedented